src/test/java/org/alfresco/util/CachingDateFormatTest.java

```diff
@@ -37,23 +38,29 @@
 {
     private final LocalDateTime REFERENCE_DATE_TIME = LocalDateTime.of(2018, 4, 1, 10, 0); //2018-04-01 at 10:00am
     private final Locale defaultLocale = Locale.getDefault();
-    
+
+    @Before
+    public void setUp() {
```

Comment:
Minor formatting.

Reply:
Hi @derekhulley yes, we were looking at the same thing, I just pushed the change with the formatting issues fixed